certification/internal/bundle/bundle.go

```diff
@@ -9,6 +9,8 @@ import (
 	"path/filepath"
 	"strings"
 
+	v1 "k8s.io/api/rbac/v1"
```

Comment:
If you're going to have an alias, something more descriptive like 'rbacv1' would be appropriate here.

Reply:
fixed 👍